scripts/do-partition.py

```diff
@@ -41,10 +41,10 @@
 DEFAULT_K = 32
 
 
-def worker(queue, basename, stop_big_traversals):
+def worker(que, basename, stop_big_traversals):
```

Comment:
Fixes https://github.com/dib-lab/khmer/issues/1257
`queue` was hiding the module `queue`